testing/docker.go

```diff
@@ -207,7 +212,7 @@ func (d *DockerContainer) portMapping(selectFirst bool, cPort int) (containerPor
 			if err != nil {
 				return 0, "", 0, err
 			}
-
+			// nolint: staticcheck
 			return uint(port.Int()), binding.HostIP, uint(hostPortUint), nil
```

Comment:
Likewise, this should be on the same line: `return uint(port.Int()), binding.HostIP, uint(hostPortUint), nil  // nolint: staticcheck`